ThirdPartyAdapters/inmobi/inmobi/src/main/java/com/google/ads/mediation/inmobi/InMobiExtrasBuilder.java

```diff
@@ -15,14 +23,26 @@ public class InMobiExtrasBuilder {
   public static final String COPPA = "coppa";
 
   @NonNull
-  public static InMobiExtras build(@Nullable Bundle mediationExtras, @NonNull String protocol) {
+  public static InMobiExtras build(@NonNull Context context, @Nullable Bundle mediationExtras, @NonNull String protocol) {
     HashMap<String, String> map = new HashMap<>();
     // Set keywords as an empty string for now.
     String keywords = "";
     if (mediationExtras != null && mediationExtras.keySet() != null) {
       for (String key : mediationExtras.keySet()) {
         map.put(key, mediationExtras.getString(key));
       }
+      //US privacy string from Mediation Extras
+      String iabUSPrivacyString = mediationExtras.getString(InMobiNetworkKeys.IAB_US_STRING);
+      if (iabUSPrivacyString != null) {
+        InMobiPrivacyCompliance.setUSPrivacyString(iabUSPrivacyString);
+      }
+    }
+
+    //US privacy string from SharedPreferences
+    SharedPreferences sharedPref = PreferenceManager.getDefaultSharedPreferences(context);
```

Comment:
Value in shared preference will take precedence over the value from Extras. Is this recommended?

Reply:
@sunilkuna5-inmobi Addressed the comment